src/main/java/HM09/Main.java

```diff
@@ -0,0 +1,25 @@
+package HM09;
+
+
+import java.util.concurrent.*;
+
+public class Main {
+    public static void main(String[] args) throws ExecutionException, InterruptedException {
+        ThreadResource resource = new ThreadResource(0);
+
+        ExecutorService executor = Executors.newFixedThreadPool(1);
+
+        Callable callable2 = new CountThread(resource, "name");
+        Callable callable3 = new CountThread(resource, "name");
+        Callable callable4 = new CountThread(resource, "name");
+
```

Comment:
I don't see any significant mistakes so... 
Since you have a constructor setting the name of CountThread then it would be nice to have some actual Name for each : ) 
Also there is some mismatch in variable names, like future works with callable2, future2 - callable3 etc.